.travis.yml

```diff
@@ -1,7 +1,10 @@
 ---
 language: node_js
 node_js:
+  - "4"
```

Comment:
Node 4 is failing, seems some missing `'use strict';`. Can you add those?

Reply:
Yeah! I'll make sure to test locally against Node 4 and fix those.